test/cypress/integration/user/transaction_details_spec.js

```diff
@@ -84,6 +84,62 @@ describe('Transactions details page', () => {
       // Email
       cy.get('.transaction-details tbody').find('tr').eq(13).find('td').eq(1).should('have.text',
         aSmartpayCharge.email)
+      cy.get('#delayed-capture').should('not.exist')
+    })
+
+    it('should display transaction details correctly when delayed capture is ON', () => {
+      const chargeWithDelayedCapture = selfServiceDefaultUser.sections.transactions.data[1]
+      const chargeDetails = selfServiceDefaultUser.sections.transactions.details_data.filter(item => item.charge_id === chargeWithDelayedCapture.charge_id)[0]
+
+      cy.visit(`${transactionsUrl}/${chargeWithDelayedCapture.charge_id}`)
+
+      // Ensure page title is correct
+      cy.title().should('eq', `Transaction details ${chargeWithDelayedCapture.reference} - System Generated test - GOV.UK Pay`)
+
+      // Ensure page details match up
+
+      // Reference number
+      cy.get('.transaction-details tbody').find('tr').first().find('td').eq(1).should('have.text',
+        chargeWithDelayedCapture.reference)
+      // Status
+      cy.get('.transaction-details tbody').find('tr').eq(2).find('td').eq(1).should('have.text',
+        capitalise(chargeWithDelayedCapture.state.status))
+      // Amount
+      cy.get('.transaction-details tbody').find('tr').eq(3).find('td').eq(1).should('have.text',
+        convertAmounts(chargeWithDelayedCapture.amount))
+      // Refunded amount
+      cy.get('.transaction-details tbody').find('tr').eq(4).find('td').eq(1).should('have.text',
+        convertAmounts(chargeDetails.refund_summary.amount_submitted))
+      // Date created
+      cy.get('.transaction-details tbody').find('tr').eq(5).find('td').eq(1).should('contain',
```

Comment:
All these `td` have id’s so you should be able to do which is maybe less brittle (and also could be faster)

`cy.get('#description').should`…

Reply:
I agree with you on that, but it looks like the test is going into a particular order and this might be important in not breaking the ordering in the future.